src/zkevm_specs/evm/execution/addmod.py

```diff
@@ -0,0 +1,61 @@
+from ..instruction import Instruction, Transition
+from ..opcode import Opcode
+from zkevm_specs.util import FQ, RLC
+
+
+# Returns 1 when a is lower than b, 0 otherwise
+def lt_u256(instruction: Instruction, a: RLC, b: RLC) -> FQ:
+    # decode RLC to bytes for a and b
+    a_lo, a_hi = instruction.word_to_lo_hi(a, True)
+    b_lo, b_hi = instruction.word_to_lo_hi(b, True)
```

Comment:
If both `a` and `b` come from stack lookup, then I think we don't need the extra check here.

Reply:
But if we do not check this, how are `a_lo`, and  `a_hip  binded to `a` then?